### PR TITLE
Integrate another bioMLmodel (biomd0000001076)

### PR DESCRIPTION
**Pull request by anuprulez**
ping @bgruening thanks!

Integrates one of the models from (https://github.com/bgruening/galaxytools/issues/1365): [BIOMD0000001076](https://www.ebi.ac.uk/biomodels/BIOMD0000001076)

The docker container is at: docker.io/anupkumar/biomodels_biomd0000001076:1. I could not push it to `quay.io/galaxy/...`

The source for building docker is at: https://github.com/anuprulez/biomodelsml-docker/tree/master/prediction_bc_mlp

### CODE REVIEW

**Comment by bgruening** [comment]
Can you look at the failing test? Anup should we move the containers to usegalaxy-eu organisation.

**Comment by anuprulez** [comment]
Ok, tests are fixed. I have moved the containers repo to https://github.com/usegalaxy-eu/docker-biomodels-ml

File: tools/biomodelsML/biomodels_BIOMD0000001076.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<tool id="biomodels_BIOMD0000001076" name="Prediction of Breast Cancer (BC)" version="@VERSION@" profile="22.05">
+    <description>proteins involved in cancer immunotherapy using molecular descriptors and Multi Layer Perceptron (MLP) neural network</description>
+    <macros>
+        <token name="@VERSION@">1</token>
+    </macros>
+    <creator>
```
Comment (bgruening) [inline tools/biomodelsML/biomodels_BIOMD0000001076.xml:6]: Is that ok for you?

File: tools/biomodelsML/biomodels_BIOMD0000001076.xml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+**Output file**
+
+Returns predicted data as a tabular file - the predictions are concatenated as a last column of the test data.
+
+        ]]>
+    </help>
+    <citations>
+        <citation type="bibtex">
+            @ARTICLE{BIOMD0000001076,
```
Comment (bgruening) [inline tools/biomodelsML/biomodels_BIOMD0000001076.xml:94]: maybe that DOI 10.1038/s41598-020-65584-y  - maybe in addition?

File: tools/biomodelsML/biomodels_BIOMD0000001076.xml (hunk 1 above)
Comment (anuprulez) [reply tools/biomodelsML/biomodels_BIOMD0000001076.xml:6]: yes, it works. Thanks a lot :)

File: tools/biomodelsML/biomodels_BIOMD0000001076.xml [hunk 3]
```diff
@@ -94,7 +94,8 @@ Returns predicted data as a tabular file - the predictions are concatenated as a
             @ARTICLE{BIOMD0000001076,
                 Author = {López-Cortés A, Cabrera-Andrade A, Vázquez-Naya JM, Pazos A, Gonzáles-Díaz H, Paz-Y-Miño C, Guerrero S, Pérez-Castillo Y, Tejera E, Munteanu CR},
                 title = {{Prediction of breast cancer proteins involved in immunotherapy, metastasis, and RNA-binding using molecular descriptors and artificial neural networks. Scientific reports , 5/ 2020 , Volume 10 , Issue 1 , pages: 8515 , PubMed ID: 32444848}},
-                url = {https://www.ebi.ac.uk/biomodels/BIOMD0000001076}
+                url = {https://www.ebi.ac.uk/biomodels/BIOMD0000001076},
```
Comment (bgruening) [inline tools/biomodelsML/biomodels_BIOMD0000001076.xml:97]: What I meant is type doi.

File: tools/biomodelsML/biomodels_BIOMD0000001076.xml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        ]]>
+    </help>
+    <citations>
+        <citation type="bibtex">
+            @ARTICLE{BIOMD0000001076,
+                Author = {López-Cortés A, Cabrera-Andrade A, Vázquez-Naya JM, Pazos A, Gonzáles-Díaz H, Paz-Y-Miño C, Guerrero S, Pérez-Castillo Y, Tejera E, Munteanu CR},
+                title = {{Prediction of breast cancer proteins involved in immunotherapy, metastasis, and RNA-binding using molecular descriptors and artificial neural networks. Scientific reports , 5/ 2020 , Volume 10 , Issue 1 , pages: 8515 , PubMed ID: 32444848}},
+                doi = {10.1038/s41598-020-65584-y}
+            }
+        </citation>
```
Comment (bgruening) [inline tools/biomodelsML/biomodels_BIOMD0000001076.xml:99]:
```suggestion
        <citation type="doi">10.1038/s41598-020-65584-y</citation>
```

This is what I was asking? Is there a good reason to use bibtex?

**Comment by anuprulez** [comment]
I see that the CI's `lxml` package is now `lxml-5.0.0-cp37-cp37m-manylinux_2_28_x86_64.whl.metadata` updated but in the previous successful run (https://github.com/bgruening/galaxytools/actions/runs/7299086010/job/19891280680), it was `lxml-4.9.4-cp37-cp37m-manylinux_2_28_x86_64.whl.metadata`. Maybe that's the cause of failing tests in `planemo lint` step? Locally, I am using the same `planemo==0.75.19` as the CI.

```
lxml.etree.XMLSchemaParseError: complex type 'Output': The content model is not determinist., line 5329
```

**Comment by bgruening** [comment]
This is a known problem due to a new lxml version. `pip install 'lxml<5'` will fix that or new Galaxy packages.

Why did you remove the creator tag?

**Comment by anuprulez** [comment]
> Why did you remove the creator tag?

I was testing to resolve the lining error. I made a few changes to find the cause of the lining error in the XML like removing citations as well. Locally, `planemo lint` worked fine. It's all there now.

> This is a known problem due to a new lxml version. pip install 'lxml<5' will fix that or new Galaxy packages.

ah, good to know :)

**Review by bgruening — APPROVED** [review]
Ready?

**Comment by anuprulez** [comment]
yes, I think so :)

**Pull request merged**